### PR TITLE
Mention Javadoc in README and remove revoked certificate checks

### LINKED ISSUE #39: Misleading revoked-certificate test
sendRevokedToTest() implies that the system is actively rejecting revoked certificates, which is not the case. The test passes, not because the certificate was revoked, but because the certificate is not signed by a trusted CA. As such, the test should probably be removed as revoked certificates should not be a part of a system of only self-signed root certificates.

### LINKED ISSUE #40: No mention of the JavaDoc in README.md
The JavaDoc can be found as an artifact in the latest workflow run (Actions).
However, the README does not mention that a JavaDoc exists for this project, which might be useful to know.

Also, it should be mentioned that you need to be logged in to download artifacts.

### PR DESCRIPTION
**Pull request by william-stacken**
Fixes #40 
Fixes #39 
@Cactu5

### CODE REVIEW

**Comment by Cactu5** [comment]
Have you looked into why the tests are failing?

**Comment by william-stacken** [comment]
@Cactu5 fixed now. I just needed to initialize it the same way as the keystore. Ready to merge?

File: decentralizediso20022/src/main/java/com/centiglobe/decentralizediso20022/util/HTTPSFactory.java [hunk 1]
```diff
@@ -90,19 +84,7 @@ public static TrustManagerFactory createTrustManager(String truststorePath, Stri
...
-        rc.setOptions(EnumSet.of(PKIXRevocationChecker.Option.PREFER_CRLS, // prefer CLR over OCSP
-                PKIXRevocationChecker.Option.SOFT_FAIL, PKIXRevocationChecker.Option.NO_FALLBACK));
-        // TODO: Remove SOFT_FAIL and change to ONLY_END_ENTITY
-        // and don't fall back to OCSP checking.
-
-        PKIXBuilderParameters pkixParams = new PKIXBuilderParameters(keystore, new X509CertSelector());
-        pkixParams.addCertPathChecker(rc);
-
-        tmf.init(new CertPathTrustManagerParameters(pkixParams));
+        tmf.init(keystore);
```
Comment (Cactu5) [inline decentralizediso20022/src/main/java/com/centiglobe/decentralizediso20022/util/HTTPSFactory.java:87]: Why was this removed?
Since the tests pass I would assume this is fine, but I would like to know some more context to this change.

Comment (william-stacken) [reply decentralizediso20022/src/main/java/com/centiglobe/decentralizediso20022/util/HTTPSFactory.java:87]: Because there is no need to check for revoked certificates if all financial institutions are root CAs that cannot revoke themselves.

**Review by Cactu5 — APPROVED** [review]
